--- a/src/tracee/tracee.h
+++ b/src/tracee/tracee.h
@@ -22,12 +22,12 @@
 #define TRACEE_H
 
 #include <sys/types.h> /* pid_t, size_t, */
-#include <sys/user.h>  /* struct user*, */
 #include <stdbool.h>   /* bool,  */
 #include <sys/queue.h> /* LIST_*, */
 #include <sys/ptrace.h>/* enum __ptrace_request */
 #include <talloc.h>    /* talloc_*, */
 #include <stdint.h>    /* *int*_t, */
 
 #include "arch.h" /* word_t, user_regs_struct, HAS_POKEDATA_WORKAROUND */
+#include <sys/user.h>  /* struct user*, */
 #include "compat.h"
 
 typedef enum {

